package.json

```diff
@@ -36,8 +36,8 @@
     "eslint-plugin-import": "^1.15.0",
     "eslint-plugin-jsx-a11y": "^2.2.2",
     "eslint-plugin-react": "^6.2.1",
-    "extract-text-webpack-plugin": "^1.0.1",
-    "file-loader": "^0.9.0",
+    "extract-text-webpack-plugin": "grrowl/extract-text-webpack-plugin#order-undefined-error",
```

Comment:
The point is that they don't fail the build with this fork. I did try and correct the issue in `header/style.css` but didn't make much progress. Practically, it won't matter unless there are styles with exactly the same level of specificity being generated in a non-deterministic order.